PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -114,9 +120,10 @@ struct HfCandidateSelectorD0 {
   }
 
   /// Conjugate-independent topological cuts
+  /// \param reconstructionType is the reconstruction type (DCAFitterN or KFParticle)
   /// \param candidate is candidate
   /// \return true if candidate passes all cuts
-  template <typename T>
+  template <int reconstructionType, typename T>
```

Comment:
is it necessary here to add this additional `int reconstructionType`? The lines in which this is used are commented now

Reply:
Hi @mfaggin, the reason to add the `int reconstructionType` is that the `processWithDCAFitterN` and `processWithKFParticle` will input different tables to the functions (`processSel`, `selectionTopolConjugate` and `selectionTopol`). The `processWithKFParticle` has the extra `HfCand2ProngKF` table.